smoketest/tests.py

```diff
@@ -625,7 +642,8 @@ def description(self):
     def __nonzero__(self):
         desired_value = self.test.value
```

Comment:
I think this line is unnecessary now (flake8 says `F841 local variable 'desired_value' is assigned to but never used`). If this is removed, I think https://github.com/usnews/smoketest/pull/30/files#diff-f2a7765a337ff9cbdb04e0dbd89c5e51L755 and the `value` argument to HeaderTest's constructor will become unneeded too.

Reply:
Nice catch. I addressed the unused variable in 796de1acae7ff94ab933604e4ae7ec3842e32ef1. I'm still unclear on the `value` code.